scss/media.scss

```diff
@@ -0,0 +1,588 @@
+      display: inline-block;
+      color: #fff;
+      font-family: "PT Sans";
+      font-weight: bold;
+      text-transform: uppercase;
+      font-size: 15px;
+      background-color: #50c3af;
+      margin-top: 67px;
+      padding-top: 16px;
+      padding-bottom: 15px;
+    }
+  }
+
+  //!HEADER
+
+  //BENFITS
+
+  .benefits{
+    background-color: #eeeeef;
+    display: flex;
+    justify-content: space-around;
+    flex-direction: row;
+    padding: 56px 93px 62px;
+    .benefit-item{
+      margin-bottom: 0px;
+      display: flex;
+      justify-content: space-between;
+      flex-direction: column;
+      align-items: center;
+      width: 24%;
+      text-align: center;
+      img{
+        display: inline-block;
+        border-style: none;
+        margin-bottom: 10px;
+      }
+      h2{
+        text-transform: uppercase;
+        font-family: "PT Sans";
+        font-weight: bold;
+        font-size: 16px;
+        position: relative;
+        color: #384048;
+      }
+      div{
+        line-height: 24px;
+        font-family: "PT Sans";
+        font-weight: normal;
+        font-style: normal;
+        font-size: 14px;
+        margin-top: 25px;
+        letter-spacing: 0.8px;
+        position: relative;
+        color: #787b80;
+        &::before{
+          top: -21px;
+          position: absolute;
+          content: "";
+          width: 120px;
+          height: 2px;
+          background-color: #e2e3e4;
+          left: 16px;
+          transform: translate(0);
+        }
+      }
+    }
+  }
+
+  //!BENEFITS
+
+
+  //Another BENEFITS
+
+  .another-benefits{
+    display: flex;
+    justify-content: space-around;
+    padding: 54px 54px;
+    flex-direction: row;
+    .another-benefits-item{
+      width: 35%;
+      img{
+        width: 100%;
+      }
+      h2{
+        color: #384048;
+        font-family: "PT Sans";
+        text-transform: uppercase;
+        font-size: 18px;
+        margin-top: 34px;
+        margin-bottom: 50px;
+        font-weight: bold;
+        position: relative;
+        &::after{
+          left: 0px;
+          bottom: -21px;
+          position: absolute;
+          height: 2px;
+          width: 240px;
+          content: "";
+          /* Permalink - use to edit and share this gradient: http://colorzilla.com/gradient-editor/#e9e9ea+0,eceded+50,f1f1f1+74,f3f3f3+100 */
+          background: #e9e9ea; /* Old browsers */
+          background: -moz-linear-gradient(left, #e9e9ea 0%, #eceded 50%, #f1f1f1 74%, #f3f3f3 100%); /* FF3.6-15 */
+          background: -webkit-linear-gradient(left, #e9e9ea 0%,#eceded 50%,#f1f1f1 74%,#f3f3f3 100%); /* Chrome10-25,Safari5.1-6 */
+          background: linear-gradient(to right, #e9e9ea 0%,#eceded 50%,#f1f1f1 74%,#f3f3f3 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+          filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#e9e9ea', endColorstr='#f3f3f3',GradientType=1 ); /* IE6-9 */
```

Comment:
https://caniuse.com/#search=gradient
You don't need to use vendor prefixes for linear gradient. It's already pretty well supported.